### ISSUE TITLE
[Security] Uploaded images are exposed through WordPress REST API

### ISSUE THREAD

**Issue by danburzo** [wpninjas/ninja-forms#3093]
- **What actions are you taking?**
  Website visitors submit a Ninja Form that includes some uploaded images.
- **What are you expecting to occur?**
  The images should not be available on `mywebsite.com/wp-json/wp/v2/media` 
- **What is occurring instead?**
  The images are available in the media list that the WordPress REST API provides.

**Comment by klhall1987** [comment]
Hey @danburzo, 

Thanks for bringing this to our attention. Does this happen if you don’t have files set to upload to the media library? Are all your media files visible there, or only files that have been uploaded with the file uploads add on?

**Comment by kjohnson** [comment]
@danburzo in my testing, I'm only seeing uploads exposed that are set to "Save to Media Library", which seems to be expected behavior for the REST API's `/media` endpoint.

To avoid the uploads being visible you can unset the setting to "Save to Media Library".

Following up with @polevaultweb via wpninjas/ninja-forms-uploads#299

**Comment by kjohnson** [comment]
This is currently known to be expected behavior, in accordance with the exceptions of the Media Library and the REST API. https://github.com/wpninjas/ninja-forms-uploads/issues/299#issuecomment-331641349

**Comment by danburzo** [comment]
@klhall1987 @kjohnson Thank you! And sorry, I had missed the GitHub notifications on this issue over the weekend. I can confirm I had the "Save to Media Library" setting enabled on the form I had used for the submissions — I'll retest with the option disabled. 

I'm not sure I have enabled it myself or if it's the default setting — in case it is the default setting, I think there should be a bit of info somewhere to make people aware of the implications, now that WP REST API ships with out-of-the-box public access to these files.
